### PR TITLE
Font updates

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

File: src/app/components/Transactions/index.tsx [hunk 1]
```diff
@@ -75,23 +76,27 @@ export const Transactions: FC<TransactionProps> = ({
       },
       {
         content: (
-          <TrimLinkLabel
-            label={transaction.hash}
-            to={RouteUtils.getTransactionRoute(transaction.hash, ParaTime.Emerald)}
-          />
+          <Typography variant="mono">
+            <TrimLinkLabel
```
Comment (csillag) [inline src/app/components/Transactions/index.tsx:80]: Wouldn't it make sense to use the Typography inside the link component?

**Review by csillag — APPROVED** [review]
Reviewed. LGTM.

File: src/app/components/Transactions/index.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/components/Transactions/index.tsx:80]: I've done it this way initially (inside the TrimLabel), but moved typography outside the component. Wasn't sure if it's really belong there. Anyway, will gather additional feedback before merging.

Comment (lubej) [reply src/app/components/Transactions/index.tsx:80]: From the usage perspective I prefer if the `Typography` is outside, and the only thing that is actually clickable is the content itself. `Typography` could have some padding, which would make "empty" space clickable.

**Review by lubej — APPROVED** [review]

File: src/styles/theme/defaultTheme.ts [hunk 2]
```diff
@@ -409,13 +410,20 @@ export const defaultTheme = createTheme({
     },
     MuiTypography: {
       variants: [
+        {
+          props: { variant: 'mono' },
+          style: () => ({
+            fontFamily: 'Roboto MonoVariable',
```
Comment (lukaw3d) [inline src/styles/theme/defaultTheme.ts:416]: I'd add a monospace fallback

**Pull request merged**